commit 4169ccd8df414cefe3611dfead9c7ef82fa49fe1
Author: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date:   Wed Jan 1 12:23:50 2014 -0800

    configure: Disable osxvideo on Leopard and earlier
    
    This also moves the "other platforms" check in OS X video to before the
    variable is read
    
    Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>

diff --git a/configure.ac b/configure.ac
index 3cb5109..b29f634 100644
--- configure.ac
+++ configure.ac
@@ -486,17 +486,17 @@ translit(dnm, m, l) AM_CONDITIONAL(USE_OSX_VIDEO, true)
 HAVE_OSX_VIDEO="no"
 AG_GST_CHECK_FEATURE(OSX_VIDEO, [OSX video], osxvideosink, [
   AC_CHECK_HEADER(OpenGL/gl.h, HAVE_OSX_VIDEO="yes", HAVE_OSX_VIDEO="no")
-])
-dnl in case header OpenGL/gl.h is found on other platforms
-case "$host" in
-  *-*darwin*)
-    dnl do nothing
-    ;;
-  *)
-    HAVE_OSX_VIDEO="no"
-    ;;
-esac
 
+  dnl in case header OpenGL/gl.h is found on other platforms (or older, unsupported OS X)
+  case "$host" in
+    *-*darwin1*)
+      dnl do nothing on Snow Leopard or newer
+      ;;
+    *)
+      HAVE_OSX_VIDEO="no"
+      ;;
+  esac
+])
 
 dnl *** Video 4 Linux 2 ***
 dnl for information about the header/define, see sys/v4l2/gstv4l2element.h
